src/app/features/cinematic/index.js

```
@@ -0,0 +1,278 @@
+      .addCase(getCinematicCategories.fulfilled, (state, action) => {
+        state.isFetching = false;
+        state.categories = action?.payload;
+      })
+      .addCase(getCinematicCategories.rejected, (state, action) => {
+        state.isFetching = false;
+        state.error = action?.payload;
+      })
+      .addCase(getCinematicByCategory.pending, (state) => {
+        state.isVideoFetching = true;
+      })
+      .addCase(getCinematicByCategory.fulfilled, (state, action) => {
```

ESLint

[error] 'action' is defined but never used.

(no-unused-vars)